src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/LearnerDataRepository.cs

```diff
@@ -56,7 +66,90 @@ public async Task Update(LearnerModel learnerModel)
 
             if (existingLearner != null)
             {
-                _dbContext.Entry(existingLearner).CurrentValues.SetValues(updatedLearner);
+                UpdateLearner(updatedLearner, existingLearner);
+            }
+        }
+
+        private void UpdateLearner(Learner updatedLearner, Learner existingLearner)
```

Comment:
Could use a basic refactor to reduce the size of the method.

Reply:
Done.